pyseq.py

```diff
@@ -876,6 +1060,40 @@ def get_sequences(source):
             seq = Sequence([item])
             seqs.append(seq)
 
-    log.debug('time: %s' % (datetime.now() - start))
-
-    return list(seqs)
+    log.debug("time: %s" %(datetime.now() - start))
+    if not stereo:
+        log.info("added sequences: %s" %(seqs))
+        return seqs
+    else:
+        seqs.sort()
```

Comment:
I was referring to the complexity of having multiple list and dicts where, I think, just a sort, some regex splits, and two lists work
```
new_seqs = []
view_seqs = []
seqs = natural_sort(seqs)
multi_view = None
while len(seqs) > 0:
    seq = seqs.pop(0)
    spl = view_regex.split(seq.format())
    # not split, it's not a multi
    if len(spl) == 1:
        new_seqs.append(seq)
    else:
        # maybe it's a view, check the next one in the list for a pair
        next_spl = regex.split(seqs[1].format()
        # check the split len and find if there's only one difference
        if len(next_spl) == len(spl) and one_diff(next_spl, spl):
            # append to the previous multiview or make a new one
            if multi_view is not None:
               multi_view.append(seq)
            else:
                multi_view = MultViewSeq(seq)
        else:
            # not match with next one, either add to previous multiview or add to single seqs
            if mutli_view is None:
                new_seqs.append(seq)
            else:
                multi_view.append(seq)
                view_seqs.append(multi_view)
                multi_view = None
```
Some thing along those lines.